packages/manager/src/components/Feedback/declarations.d.ts

```diff
@@ -0,0 +1,6 @@
+declare namespace JSX {
+  interface IntrinsicElements {
+    'opc-feedback': any;
+  }
+}
+declare module '@one-platform/opc-feedback/dist/opc-feedback';
```

Comment:
Hi @nikhitamore , do we need this `@one-platform/opc-feedback/dist/opc-feedback` ! As we are not using the service !

Reply:
my apologies, let me remove this one, we don't need it, its remaining from previous integration

Reply:
yes we need this, its not for service, its to use ops-feedback in our Nextapp

Reply:
opc-feedback is a web component it can be used with any web api

Reply:
<img width="1624" alt="image" src="https://github.com/spaship/spaship/assets/26894183/09de7c8b-fd5c-4341-9548-995d1be0896c">
we get this error if I remove those declarations.d.ts 
`Property 'opc-feedback' does not exist on type 'JSX.IntrinsicElements'.ts(2339)`